### PR TITLE
CVE-2020-1706 init

### PR DESCRIPTION
**Pull request by rehakm**

### CODE REVIEW

File: 2020/1xxx/CVE-2020-1706.json [hunk 1]
```diff
@@ -4,15 +4,71 @@
...
+                "refsource": "CONFIRM"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "It has been found that multiple containers modify the permissions of /etc/passwd to make them modifiable by users other than root. An attacker with access to the running container can exploit this to modify /etc/passwd to add a user and escalate their privileges. This CVE is specific to the openshift/apb-tools-container."
```
Comment (cve-team) [inline 2020/1xxx/CVE-2020-1706.json]: The description needs to include the versions affected as well. Thanks!

Comment (rehakm) [reply 2020/1xxx/CVE-2020-1706.json]: Added version to description, thanks!

Comment (cristina479) [reply 2020/1xxx/CVE-2020-1706.json]: Hello,

Please include the affected versions in the description.

Comment (rehakm) [reply 2020/1xxx/CVE-2020-1706.json]: Versions included.

**Comment by rehakm** [comment]
Hello cve team this should now be ok, would you please review?

**Pull request merged**